redwoods-insurance-salesforce/.prettierrc

```diff
@@ -10,6 +10,10 @@
         {
             "files": "*.{cmp,page,component}",
             "options": { "parser": "html" }
+        },
+        {
+            "files": "*.*-meta.xml",
```

Comment:
@codefriar is that parser override really needed?
The file extension is xml so that should work by default.

Reply:
I was getting errors on `prettier --list-different` when I ran it without the override in place. Specifically it would tell me that it couldn't locate the parser. This fixed it.

Reply:
CI checks are running on the update that strips this override.